### DISCUSSION TITLE
Tried to copy a database and broke project

### DISCUSSION THREAD

**Discussion by tilmann** [supabase/supabase#5842] [Questions]
I was trying to get data from my BETA database into my DEV database. 

In the process I ran some SQL scripts in the supabase UI. I seem to have lost all supabase schemas. 
<img width="244" alt="2022 03 07 - 12-03-23 - Supabase" src="https://user-images.githubusercontent.com/7314399/157038590-a01066da-4ba0-4faf-8b77-e79f3d03855b.png">

If I try to add a `storage` bucket I see the following error message: `Migration failed. Reason: An error occurred running 'create-migrations-table'. Rolled back this migration. No further migrations were run. Reason: no schema has been selected to create in`

Is there an option to reset the whole project?

**Comment by tilmann** [comment]
I was able to fix the storage problem by running the init script I found here: https://github.com/supabase/supabase/blob/master/docker/volumes/db/init/00-initial-schema.sql
(just comment the lines out that stop executing the script)

But I still cannot use auth. I always get the error message: 
```
{
  "message":"An invalid response was received from the upstream server"
}
```

https://yuwkdzutijwagbxpthie.supabase.co/auth/v1/authorize?provider=google

Any ideas how to fix it?

**Reply by colestriler** [reply]
This one resolved itself for me.
